wallet/src/account/operations/transaction/prepare_output.rs

```diff
@@ -116,12 +119,7 @@ impl AccountHandle {
 
                     // Calculate the minimum storage deposit to be returned
                     let min_storage_deposit_return_amount =
-                        BasicOutputBuilder::new_with_minimum_storage_deposit(rent_structure.clone())?
-                            .add_unlock_condition(UnlockCondition::Address(AddressUnlockCondition::new(
-                                Address::try_from_bech32(options.recipient_address.clone())?.1,
-                            )))
-                            .finish_output(token_supply)?
-                            .amount();
+                        minimum_storage_deposit_basic_output(&rent_structure, &None, token_supply)?;
```

Comment:
Is that just for convenience or did it not return the same value?

Reply:
Just convenience